resources/views/pages/user/[user]/comments.blade.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use App\Models\User;
+use Illuminate\View\View;
+
+use function Laravel\Folio\{middleware, name, render};
+
+middleware(['auth', 'can:viewAny,' . User::class]);
```

Comment:
```suggestion
middleware(['can:viewAny,' . User::class]);
```

If I'm not mistaken, these are also currently public-facing.

Reply:
done